src/app/_services/group.service.ts

```diff
@@ -43,7 +43,7 @@ export class GroupService {
             tags = []
         }
         if (!size) {
-            size = 48
+            size = 25
```

Comment:
Is there a way to make this a constant somewhere so if we have to set it again we only have to change it in one place?

Reply:
Good question. Let me try.